### PR TITLE
Add cron GitHub Action to run security checks every day

### LINKED ISSUE #21199: CI: Run symfony security check
#### Is your feature request related to a problem?

Some actions are performed too late in the build process, resulting in delays. We can leverage CI to automate these actions nightly.

#### Describe the solution you'd like

Add the following tests to nightly tests in order to verify that no dependencies contain known security vulnerabilities:

```
php bin/console security:check
```

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add cron GitHub Action to run security checks every day
| Type?             | new feature
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/21199 and https://github.com/PrestaShop/PrestaShop/issues/22924
| How to test?      | 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23143)

### CODE REVIEW

**Comment by matks** [comment]
Run on push on my fork => https://github.com/matks/PrestaShop/runs/1832770712?check_suite_focus=true

(result is red... as expected 😅  we _really_ have unsecure dependencies)

File: .github/workflows/daily-checks.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Cron Daily Checks
```
Comment (matks) [inline .github/workflows/daily-checks.yml:1]: I created a generic name as I think we can fit many of these checks in there. Like license headers check, security checks...

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Can't be accepted without any definition of license for `pplotka/local-php-security-checker-github-actions`.

**Comment by matks** [comment]
> Can't be accepted without any definition of license for `pplotka/local-php-security-checker-github-actions`.

Nice catch! I will ask the author if he's willing to add one. If he does not want I will do the necessary boilerplate for this to work

**Comment by paras-malhotra** [comment]
@matks you can consider the [Enlightn Security Checker](https://github.com/enlightn/security-checker). It's almost a drop-in replacement and has no licensing issues (licensed under MIT). I'm the author of this package.

The issue with the Local PHP Security Checker is that it's licensed under AGPL, so I reckon even the Github action (that distributes the source) must be licensed under AGPL.

**Comment by matks** [comment]
Hello, it seems this PR is now stale.

I close this PR, following our [review guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/) "Pull Requests may be closed after 30 days of inactivity".

**Pull request closed without merging**